screens/buyer_search.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.rectangle_orange} />
+        <View style={styles.rectangle_white} />
+        <View style={styles.rectangle_LBlue} />
+        <View style={styles.rectangle_DBlue} />
+        <Text style = {styles.header}>Let's{"\n"}shopU!</Text>
```

Comment:
You have to escape the apostrophe so this red doesn't show up.

Use this [link](https://github.com/jennyengelman/PongMates/blob/master/screens/FoundAPartner.js#L20) as reference to see how to escape the apostrophe